Examples/ArcGISToolkitExamples/Misc/ARStatusViewController.storyboard

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14490.70" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="4p9-jS-sat">
+    <device id="retina6_1" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14490.49"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Status View Controller-->
+        <scene sceneID="gug-M0-qcn">
+            <objects>
+                <tableViewController id="4p9-jS-sat" customClass="ARStatusViewController" customModule="ArcGISToolkitExamples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="static" style="plain" separatorStyle="none" allowsSelection="NO" rowHeight="22" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" id="bjp-Dc-dPX">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <sections>
+                            <tableViewSection id="Nc7-BF-joG">
+                                <cells>
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" id="MqH-RN-jot" userLabel="Tracking State">
```

Comment:
The cells look fine in portrait, but rotating to landscape reveals a problem:

![image](https://user-images.githubusercontent.com/2257493/62501331-3b0e8080-b79f-11e9-8213-150886034be1.png)

An easy solution would be to just use Right Detail style cells.  Here's the table with the Translation Factor row updated to use a non-custom cell:

![image](https://user-images.githubusercontent.com/2257493/62501464-e6b7d080-b79f-11e9-920a-7e4407ea4a92.png)

Reply:
I have never seen that; it works OK with all orientations on my two devices.  The `accessoryView` detail labels are set up to be right-justified;  they do not have any constraints associated with them because of how they are setup.

What device/OS version are you running on?

Reply:
I'm using an iPhone XR simulator running iOS 12.4. The issue is that the rows are being inset from the safe area on the right. The detail labels on the right side of the cell are sized from the storyboard and, due to their width, they are pushing the labels on the left further to the left.

Reply:
Ok, got it.  Made the changes, including changing the cells to Right Detail cells and adjusted the constraints to the safe area.